backend/fms_core/resources/sample.py

```diff
@@ -35,7 +35,7 @@ class SampleResource(GenericResource):
     tissue_source = Field(attribute='tissue_source', column_name='Tissue Source')
     concentration = Field(attribute='concentration', column_name='Conc. (ng/uL)', widget=DecimalWidget())
     depleted = Field(attribute='depleted', column_name='Source Depleted')
-    creation_date = Field(attribute='reception_date', column_name='Reception Date', widget=DateWidget())
+    creation_date = Field(attribute='creation_date', column_name='Reception Date', widget=DateWidget())
```

Comment:
what about this field in the single Sample creation form? 
if the user must enter it, we have to add the field manually to the form, and collect that information.

Reply:
Not sure what you mean here.